README.md

```diff
@@ -47,6 +42,4 @@ It assumes you have the Travis CLI installed and you are already logged into Tra
 
 It is worth noting that it is possible to run the PR script locally by setting the environment variables. For example:
 
- `TRAVIS=true TRAVIS_REPO_SLUG=nhsuk/connecting-to-services TRAVIS_PULL_REQUEST=<PR number of an image which still exists>  RANCHER_TEMPLATE_NAME=c2s-pharmacy-finder SPLUNK_HEC_TOKEN=<splunk token> GITHUB_ACCESS_TOKEN=<personal access token> RANCHER_ENVIRONMENT=<rancher env name> ./rancher_pr_deploy.sh`
-
- If testing changes to the rancher template during development  RANCHER_TEMPLATE_BRANCH_NAME can also be set (see above for details).
+ `TRAVIS=true TRAVIS_REPO_SLUG=nhsuk/connecting-to-services TRAVIS_PULL_REQUEST=<PR number of an image which still exists>  GITHUB_ACCESS_TOKEN=<personal access token> RANCHER_ENVIRONMENT=<rancher env name> ./deploy.sh`
```

Comment:
True. I guess there's nothing to deploy in this repo anymore though. Previously you passed in the RANCHER_TEMPLATE_NAME, but as this PR removes them I guess this isn't needed anymore. I'll update the README to state this *can* be run in an application repo.